contrib/capitalone_dataprofiler_expectations/capitalone_dataprofiler_expectations/rule_based_profiler/data_assistant/data_profiler_structured_data_assistant.py

```diff
@@ -7,27 +7,27 @@
     DataProfilerColumnDomainBuilder,
 )
 
-from great_expectations.rule_based_profiler.config import ParameterBuilderConfig
-from great_expectations.rule_based_profiler.data_assistant import DataAssistant
-from great_expectations.rule_based_profiler.data_assistant_result import (
+from great_expectations.experimental.rule_based_profiler.config import ParameterBuilderConfig
```

Comment:
Moved RBP into `experimental` since it is referenced there by `metrics_repository`